cmd/puller/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"flag"
+	"github.com/kubeflow/kfserving/pkg/puller"
+	"log"
+)
+
+var (
+	modelDir = flag.String("model_dir", "/tmp","directory for multi-model config files")
```

Comment:
Can this variable be moved to [here](https://github.com/kubeflow/kfserving/blob/master/pkg/constants/constants.go), so InferenceService's controller can mount the configMap to the same location for MMS InferenceService?

Reply:
hmm so this is a flag variable, so its specific to the controller. I believe we should leave it here. The variable that we use will be specified when defining the controller. 

i.e. 
```yaml
- args:
- --model_dir
- <my_shared_variable>
```